snippets/cpp.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Decided to duplicate these snippets between languages as I'd imagine they'll start to diverge.  but easy to just use one file as well